pom.xml

```diff
@@ -1113,6 +1125,16 @@
         <groupId>org.scalatestplus</groupId>
         <artifactId>mockito-4-11_${scala.binary.version}</artifactId>
         <version>3.2.16.0</version>
+        <exclusions>
```

Comment:
Are these exclusions needed, if we're manually managing the dependency version?

Reply:
I think it's unnecessary, but I think we should copy 
```
<dependency>
      <groupId>net.bytebuddy</groupId>
      <artifactId>byte-buddy</artifactId>
      <scope>test</scope>
    </dependency>
    <dependency>
      <groupId>net.bytebuddy</groupId>
      <artifactId>byte-buddy-agent</artifactId>
      <scope>test</scope>
    </dependency>
```
 to the module using `mockito-core` because there is just configuring `dependencyManagement` instead of `dependencies`

Reply:
Yeah it feels like one or the other of these changes is needed

Reply:
This is done. @srowen @LuciferYang